Lecture6(GithubAPI)/Component/CardComponent.js

```diff
@@ -10,8 +10,8 @@ const CardComponent = ({ details }) => {
     avatar_url,
   } = details;
   return (
-    <div className="col-sm-6 col-md-4 col-lg-3 col-xl-3 mt-4">
-      <div className="card" style={{ width: "11rem" }}>
+    <div className="col-sm-6 col-md-4 col-lg-3 col-xl mt-4">
+      <div className="card">
         <img src={avatar_url} className="card-img-top img-fluid" alt="" />
```

Comment:
give a alt text. It is good for accessibility.

Reply:
done